src/main/java/teammates/storage/sqlapi/FeedbackResponseCommentsDb.java

```diff
@@ -122,8 +124,8 @@ public void updateGiverEmailOfFeedbackResponseComments(String courseId, String o
             return;
         }
 
-        List<FeedbackResponseComment> responseComments =
-                getFeedbackResponseCommentEntitiesForGiverInCourse(courseId, oldEmail);
+        List<FeedbackResponseComment> responseComments = getFeedbackResponseCommentEntitiesForGiverInCourse(courseId,
```

Comment:
Could you revert all the whitespace and formatting changes in this file? Was this to fix a lint error?

Reply:
HAH my bad apologies for the mistake but great catch nontheless